Modules/Scripted/DataProbe/DataProbeLib/SliceViewAnnotations.py

```diff
@@ -216,6 +218,7 @@ def onSliceViewAnnotationsCheckBox(self):
     settings = qt.QSettings()
     settings.setValue('DataProbe/sliceViewAnnotations.enabled', self.sliceViewAnnotationsEnabled)
     settings.setValue('DataProbe/sliceViewAnnotations.scalarBarEnabled', self.scalarBarEnabled)
+    settings.setValue('DataProbe/sliceViewAnnotations.scalarBarBackgroundSelected', self.scalarBarBackgroundSelected)
```

Comment:
instead of a true/false flag, please save self.scalarBarSelectedLayer value directly